script/cargo.py

```diff
@@ -98,7 +103,7 @@ def build(args):
         run_cargo('clean', args)
 
     try:
-        run_cargo('build', args)
+        run_cargo('rustc', args)
```

Comment:
What does this change do?

Reply:
rustc has more options than build https://doc.rust-lang.org/cargo/commands/cargo-rustc.html

Reply:
Ok. The bug also talks about aligning with upstream's build. It would be good to have a comment here why we're using the less common command, e.g.

> # Invoke `cargo rustc` so we can pass additional linker flags on the command line.

Reply:
my understanding is that `cargo rustc` is basically a more flexible version os `cargo build` - https://stackoverflow.com/questions/60577598/whats-the-difference-between-cargos-build-and-rustc-commands
I think it's only less common because it's newer and you only need it if you want to pass args that `cargo build` doesn't understand